### ISSUE TITLE
[0.9.2.0 beta staging-1860]Building and occupancy conflict

### ISSUE THREAD

**Issue by IcedForge** [StrangeLoopGames/EcoIssues#19555]
Right now we have a new issue of placing building objects using the hammer cannot replace plants / growths as they previously did
and after checking it around i found #14968 is this something that should be fixed? It is pretty frustrating trying to build something and it looks like this ![Screenshot_25.jpg](https://images.zenhubusercontent.com/5f5a7fa59878ed957a861886/fd70e271-eb7a-4be1-84ea-4872463e21ac)
And this including the rapid random plant generation means that you keep having to remove brush and plants all over a LOT when building larger things or in bigger areas such as roads, large stockpiles and so on.
Ending you in a constant run to kill plant, move back pickup item hot select and adjust placement where you want it try to place it after taking your time making sure its in the correct place and then plants popup again and you need to redo this once more.

So i would rather see this fixed for now and possibly split all growing plants / brush into different size categorization of Large, Medium, Small plants?

So when placing objects they can replace and kill small plants without any added effort making for quality of life and still forcing you to clear overgrowth with shovel, sickle/scythe, axe or machete.
Trying to place an object on top of a plant should also net its own error message saying something like "Cannot place this here, make sure there is no larger plants in the way"

**Comment by SlayksWood** [comment]
I guess after this changes https://github.com/StrangeLoopGames/EcoIssues/issues/19066

**Comment by WeaselDog** [comment]
They should still be able to build on places with plants.

**Comment by SlayksWood** [comment]
This definitely works better, but I would do the same behavior for everything. However, perhaps we should stop at this.
So it deletes plant, ignores solid blocks and is blocked with other objects like vehicles and craft stations:
![image](https://user-images.githubusercontent.com/45708377/102885034-c5728580-4463-11eb-9b78-948e655229c9.png)
![image](https://user-images.githubusercontent.com/45708377/102884964-a247d600-4463-11eb-8ddf-2b96f7fb3075.png)
![image](https://user-images.githubusercontent.com/45708377/102885243-197d6a00-4464-11eb-892a-fdfd2aafa408.png)
![image](https://user-images.githubusercontent.com/45708377/102885293-2b5f0d00-4464-11eb-9544-1fe45ca4a574.png)

Everything works logically until we take a tree.

- [ ] It's blocked by a big tree:
![image](https://user-images.githubusercontent.com/45708377/102885355-49c50880-4464-11eb-8740-fbc982d3db32.png)

- [ ] But it ignores stump and wood pulp:
![image](https://user-images.githubusercontent.com/45708377/102885422-62cdb980-4464-11eb-9216-5b7374e0d101.png)

- [ ] And it also ignores a little tree:
![image](https://user-images.githubusercontent.com/45708377/102885503-7f69f180-4464-11eb-92c5-ec17b043fd00.png)

**Comment by SlayksWood** [comment]
@WeaselDog @MazonDel @beastyn @StanKryvenko 
What do you think about this?

**Comment by IcedForge** [comment]
I believe it should ignore blocks it cannot destroy/remove ( like trees ) but for any other plant / vegitation it needs to have a specific civic point like (Destroy not harvest etc so it does not interfere with laws and such )
